src/main/java/codesquad/springcafe/article/Article.java

```diff
@@ -0,0 +1,35 @@
+package codesquad.springcafe.article;
+
+public class Article {
+
+    private long articleId = -1;
```

Comment:
```suggestion
    private Long articleId = -1;
```
왠지 래퍼를 사용하는 게 요즘 추세이니 참고하세요 ^^